hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerInfo.java

```diff
@@ -98,7 +101,7 @@
     this.pipelineID = pipelineID;
     this.usedBytes = usedBytes;
     this.numberOfKeys = numberOfKeys;
-    this.lastUsed = Time.monotonicNow();
```

Comment:
If you change the lastUsed and stateEnterTime to lnstant type, the code will be much simpler and clean as @elek  suggested.